packer/Makefile

```diff
@@ -6,8 +6,13 @@
 #
 ################################################
 
+ifneq (, $(findstring -DMODULE_OPTEE_GUEST, $(EVMM_CMPL_FLAGS)))
```

Comment:
this is not good. can we export CC = clang in optee.cfg, and check if CC has value here?

Reply:
can we use below?
ifneq (, $(CLANG_BINDIR))
export CC = clang
else 
export CC = gcc
endif